src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -138,8 +138,11 @@ foam.CLASS({
     },
     {
       class: 'FObjectProperty',
-      of: 'foam.comics.v2.CRUDActionsAuth',
-      name: 'CRUDActionsAuth'
+      of: 'foam.comics.v2.CRUDActionsPredicate',
+      name: 'CRUDPermission',
```

Comment:
property names should not begin with upper case. 
```suggestion
      name: 'crudPermission',  
```